ports/icu/portfile.cmake

```diff
@@ -69,32 +68,38 @@ vcpkg_copy_pdbs()
 if (ENABLE_TOOLS)
     # Rearrange location of tools
     set(TOOLS
-        derb
         genbrk
-        genccode
         gencfu
-        gencmn
         gencnval
         gendict
-        gennorm2
         genrb
-        gensprep
         gentest
         pkgdata
         makeconv
-        icupkg
         icuinfo
     )
+    set(STOOLS
+        genccode
+        gencmn
+        gennorm2
+        gensprep
+        icupkg)
 
     set(TOOL_EXTENSION .exe)
 
     file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/icu)
 
     foreach (tool ${TOOLS})
         # Remove debug versions
-        file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin/${tool}${TOOL_EXTENSION})
+        file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/bin64/${tool}${TOOL_EXTENSION})

```

Comment:
This part will fail when building anything outside of x64.

Reply:
Updated to not suffix the path and change back to just bin here.